build.gradle

```diff
@@ -322,5 +331,48 @@ task buildMap(type: JavaExec, dependsOn: 'build') {
     classpath = project.sourceSets.main.runtimeClasspath
 }
 
+task codeCoverageReport(type: JacocoReport) {
+    executionData fileTree(project.rootDir.absolutePath).include("**/build/jacoco/*.exec")
+    sourceSets sourceSets.main
+    subprojects.each {
+        sourceSets it.sourceSets.main
+    }
+    // excludes = [*]
+    // includes = ['src/team3player/**']
+    afterEvaluate {
+        classDirectories.setFrom(files(classDirectories.files.collect {
+            fileTree(dir: it, include: '**/team3player/**')
+        }))
+    }
+    reports {
+        html.enabled true
+    }
+}
+
+codeCoverageReport.dependsOn {
+    subprojects*.test
+}
+
+task testCodeCoverage(type: JacocoCoverageVerification) {
+    executionData fileTree(project.rootDir.absolutePath).include("**/build/jacoco/*.exec")
+    sourceSets sourceSets.main
+    subprojects.each {
+        sourceSets it.sourceSets.main
+    }
+    violationRules {
+        rule {
+            limit {
+                minimum = 0.1
```

Comment:
This is where the %10 coverage requirement comes from. If we want to change %10 to %20 we could just change minimum to 0.2